Backpack/src/main/java/net/skyscanner/backpack/calendar/view/BpkCalendarLabelsViewModel.kt

```diff
@@ -38,10 +43,22 @@ internal class BpkCalendarLabelsViewModel(
 
   private var labels: Map<LocalDate, BpkCalendarLabelViewModel> = emptyMap()
 
+  private val placeholder = BpkCalendarLabelViewModel(
```

Comment:
We're falling back to this placeholder if the labels is not available. The main reason is that drawing text on Canvas and drawing text using TextLayout requires different coordinates. We don't want to calculate the coordinates here or pass fallback coordinates to method each time, so we just use fallback object.